benches/sync_mpsc.rs

```diff
@@ -179,22 +163,39 @@ fn uncontented_unbounded(c: &mut Criterion) {
     });
 }
 
-criterion_group!(
-    create,
-    create_1_medium,
-    create_100_medium,
-    create_100_000_medium
-);
-
-criterion_group!(send, send_medium, send_large);
-
-criterion_group!(
-    contention,
-    contention_bounded,
-    contention_bounded_full,
-    contention_unbounded,
-    uncontented_bounded,
-    uncontented_unbounded
-);
-
-criterion_main!(create, send, contention);
+fn group_create_medium(c: &mut Criterion) {
+    let mut group = c.benchmark_group("create_medium");
+    create_medium::<1>(&mut group);
+    create_medium::<100>(&mut group);
+    create_medium::<100_000>(&mut group);
+    group.finish();
+}
+
+fn group_send(c: &mut Criterion) {
+    let mut group = c.benchmark_group("send");
+    send_data::<Medium, 1000>(&mut group, "Medium");
+    send_data::<Large, 1000>(&mut group, "Medium");
```

Comment:
We are not allowed use a same name in the one benchmark group. Do you mean `"Large"` here?
Wouldn't it be better for us to use lowercase words here, `"medium"` and `"large"`?